contracts/modules/token/TokenController.sol

```diff
@@ -21,8 +21,8 @@ contract TokenController is ITokenController, LockHandler, MasterAwareV2 {
 
   address public _unused_token;
   address public _unused_pooledStaking;
-  address public _unused_minCALockTime;
-  address public _unused_claimSubmissionGracePeriod;
+  uint public _unused_minCALockTime;
```

Comment:
the fact that it was an address before was strange originally in v1 it used to be uint